### PR TITLE
Jekyll Documentation Site

### PR DESCRIPTION
**Pull request by rickwysocki**
I wasn't sure if you wanted this pushed, so feel free to ignore it if now. This is a super-basic Jekyll install using the jekyll-docs theme, if you'd like to take a look at it. If it looks good, I'm happy to set up a structure for the documentation on the site based on your suggestions.

### CODE REVIEW

**Review by iamdavidmt — COMMENTED** [review]
Think I read we needed the gh pages gem? but unsure. The domain is docs.eco-infra.com

**Comment by rickwysocki** [comment]
I'm not sure off the top of my head, but I'll double check. I'm traveling this weekend but will follow up early next week.

**Comment by rickwysocki** [comment]
Hey there: So, to set up GH Pages, you first need to set a publishing source under Settings > Pages: https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site 

At that point, once you integrate/setup Jekyll, the site should automatically build within the folder you select.

File: docs/_config.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# in the templates via {{ site.myvariable }}.
+
+title: EcoInfra
+email: your-email@example.com
+description: >- # this means to ignore newlines until "baseurl:"
+  Write an awesome description for your new site here. You can edit this
+  line in _config.yml. It will appear in your document head meta (for
+  Google search results) and in your feed.xml site description.
+baseurl: "" # the subpath of your site, e.g. /blog
+url: "" # the base hostname & protocol for your site, e.g. http://example.com
```
[the remark this answers is not in the document]
Comment (rickwysocki) [reply docs/_config.yml:28]: Yes, this is correct, but you also need to set up GH pages in settings (see below).

**Comment by iamdavidmt** [comment]
@rickwysocki done, I've set the source to /docs/ see the current /docs/ dir and docs.eco-infra.com. I say we just merge this (if you could resolve conflicts please) and debug any issues.

**Comment by rickwysocki** [comment]
Hi there: Sorry this took a minute but I'm pretty sure I've reviewed the conflicts and it should be good now.

**Comment by iamdavidmt** [comment]
On build: https://github.com/eco-infra/ecoinfra/issues/31

**Comment by rickwysocki** [comment]
Pulling these changes now to look at this locally.

**Pull request merged**